bcda/testConstants/constants.go

```diff
@@ -0,0 +1,6 @@
+package testConstants
+
+const DEVACOUUID = "0c527d2e-2e8a-4808-b11d-0fa06baf8254"
+const SMALLACOUUID = "3461C774-B48F-11E8-96F8-529269fb1459"
+const MEDIUMACOUUID = "C74C008D-42F8-4ED9-BF88-CEE659C7F692"
+const LARGEACOUUID = "8D80925A-027E-43DD-8AED-9A501CC4CD91"
```

Comment:
Very necessary.  Thank you for doing that.

Reply:
I just couldn't keep track of it all in my head.  We probably need a ticket to actually start using all of these.